velox/exec/Exchange.h

```diff
@@ -171,6 +171,11 @@ class ExchangeQueue {
     checkComplete();
   }
 
+  void closeLocked() {
+    std::deque<std::unique_ptr<SerializedPage>> queue;
+    queue = std::move(queue_);
```

Comment:
Can we just call queue_.clear() any consideration for the move? Thanks!

Reply:
+1